package.json

```diff
@@ -24,6 +24,7 @@
     "which": "^1.0.9"
   },
   "scripts": {
+    "preinstall": "npm install tsd && tsd update",
```

Comment:
Explicit path to `node_modules/.bin` isn't needed since `npm` prepends it to `PATH`. See [nmp scripts doc](https://docs.npmjs.com/misc/scripts#path). You can also see that this PR's build is green on Travis.